### PR TITLE
Remuneration policy

### LINKED ISSUE #273: Publish legal info about Foundation (KvK registration number, etc)
This includes:
- [x] de officiële naam van de instelling
- [x] het nummer dat aan de instelling is toegekend bij inschrijving in het handelsregister (RSIN)
- [x] het post- of bezoekadres, dan wel het telefoonnummer, dan wel het e-mailadres van de instelling
- [x] de doelstelling volgens de regelgeving van de instelling
- [x]  de hoofdlijnen van het actuele beleidsplan van de instelling
- [ ] de bestuurssamenstelling, het beloningsbeleid en de namen van de bestuurders

The official government list is here: https://www.belastingdienst.nl/wps/wcm/connect/bldcontentnl/belastingdienst/zakelijk/bijzondere_regelingen/goede_doelen/algemeen_nut_beogende_instellingen/gegevens_van_een_anbi_publiceren_op_een_internetsite/gegevens_van_een_anbi_publiceren_op_een_internetsite

De uitzonderingen zijn niet van toepassing.

### PR DESCRIPTION
**Pull request by ElenaFdR**
This fixes #273.

This draft PR contains:
- [ ] Dutch in need of translation
- [x] a question about intern compensation we need to factcheck (edit: removed and captured as #330)

@MirjamvT Please update the Dutch and factcheck everything!

### CODE REVIEW

File: organization/remuneration-policy.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+type: resource
+---
+
+# Remuneration policy
+
+This explains the Foundation for Public Code's approach to remuneration.
```
Comment (ElenaFdR) [inline organization/remuneration-policy.md]: I'd like to add a statement about our view on free labor on the open source community, and where we've specifically designed the Foundation to either encourage people to be paid for their work for the community or deliberately encourage volunteers.
(To address the issues from Ashe Dryden's post on OSS and unpaid labor: https://www.ashedryden.com/blog/the-ethics-of-unpaid-labor-and-the-oss-community)

File: organization/remuneration-policy.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Staff, contractors and interns
+
+Foundation staff salaries are competitive as well as in line with the social mission of the organization.
+
+> Interns may or may not be compensated, depending on whether they're part of a work placement scheme or have an external stipend. We believe strongly that everyone who works for us should receive a living wage. (FACTCHECK?)
```
Comment (ElenaFdR) [inline organization/remuneration-policy.md]: @MirjamvT Factcheck here!

File: organization/remuneration-policy.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+The Foundation will reimburse reasonable expenses for staff, contractors and interns.
+
+## Board of directors
+
+> Translate this info: Bestuurders krijgen alleen een vergoeding voor gemaakte onkosten. Ze ontvangen geen vacatiegeld. Gemaakte onkosten zijn kosten die bestuursleden redelijkerwijs hebben door hun rol als bestuurder bij de Foundation.
+
+Members of the Board of directors are not paid for their work as directors. The Foundation will reimburse directors for expenses incurred as a result of their role as director of the Foundation.
+
+> Do directors get different expenses reimbursed than non-directors?
```
Comment (ElenaFdR) [inline organization/remuneration-policy.md]: @MirjamvT Directors' expenses were worded differently from non-directors; is this significant?

File: organization/remuneration-policy.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+## Definition of reasonable expenses
+
+We reimburse reasonable expenses [incurred while working for the Foundation] OR [while carrying out work authorised by the Foundation] for everyone who works with us.  [mits het uitgaven in opdracht van de Foundation betrof].  
+
+We consider reasonable expenses to be:
+- costs incurred while working towards the [Foundation's mission] (voorgeschoten uitgaven voor het realiseren van het doel van de stichting)
+- public transport incurred during business trips: flights (economy class only), train (2nd class only), bus or other local public transport, [OV bike](https://www.ns.nl/en/door-to-door/ov-fiets) or taxis
+- private transport used for business trips: gereden kilometers in private vehicles (eg cars, motorcycles or bicycles) for business trips
+
+[Submit expenses using this form].
```
Comment (ElenaFdR) [inline organization/remuneration-policy.md]: Add link to the form once the form is live.

File: organization/remuneration-policy.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+> Interns may or may not be compensated, depending on whether they're part of a work placement scheme or have an external stipend. We believe strongly that everyone who works for us should receive a living wage. (FACTCHECK?)
+
+The Foundation will reimburse reasonable expenses for staff, contractors and interns.
+
+## Board of directors
+
+> Translate this info: Bestuurders krijgen alleen een vergoeding voor gemaakte onkosten. Ze ontvangen geen vacatiegeld. Gemaakte onkosten zijn kosten die bestuursleden redelijkerwijs hebben door hun rol als bestuurder bij de Foundation.
+
+Members of the Board of directors are not paid for their work as directors. The Foundation will reimburse directors for expenses incurred as a result of their role as director of the Foundation.
```
Comment (ElenaFdR) [inline organization/remuneration-policy.md]: Dutch original: Bestuurders krijgen alleen een vergoeding voor gemaakte onkosten. Ze ontvangen geen vacatiegeld. Gemaakte onkosten zijn kosten die bestuursleden redelijkerwijs hebben door hun rol als bestuurder bij de Foundation.

File: organization/remuneration-policy.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+> Do directors get different expenses reimbursed than non-directors?
+
+## Volunteers
+
+Volunteers are not paid for their work for the Foundation. We will reimburse reasonable expenses.
+
+## Definition of reasonable expenses
+
+We reimburse reasonable expenses [incurred while working for the Foundation] OR [while carrying out work authorised by the Foundation] for everyone who works with us.  [mits het uitgaven in opdracht van de Foundation betrof].
```
Comment (ElenaFdR) [inline organization/remuneration-policy.md]: @MirjamvT I'm uncertain of the best translation here. The Dutch original was "mits het uitgaven in opdracht van de Foundation betrof". How precise is 'in opdracht van'? "incurred on behalf of the Foundation?"

File: organization/remuneration-policy.md (hunk 1 above)
Comment (bvhme) [reply organization/remuneration-policy.md]: Great, seems like an awesome issue :)

**Comment by bvhme** [comment]
This should be a draft PR

File: organization/remuneration-policy.md (hunk 1 above)
Comment (ElenaFdR) [reply organization/remuneration-policy.md]: Done! #330

File: organization/remuneration-policy.md (hunk 2 above)
Comment (ElenaFdR) [reply organization/remuneration-policy.md]: Moved question of intern compensation to #330; deleted draft text here.

File: organization/remuneration-policy.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+Members of the Board of directors are not paid for their work as directors. The Foundation will reimburse directors for expenses incurred as a result of their role as director of the Foundation.
+
+## Volunteers
+
+Volunteers are not paid for their work for the Foundation. We will reimburse reasonable expenses.
+
+## Definition of reasonable expenses
+
+We reimburse reasonable expenses [incurred while working for the Foundation] OR [while carrying out work authorised by the Foundation] for everyone who works with us.
```
Comment (ElenaFdR) [inline organization/remuneration-policy.md]: alternative wording: while carrying out work authorised by the Foundation

File: organization/remuneration-policy.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+Volunteers are not paid for their work for the Foundation. We will reimburse reasonable expenses.
+
+## Definition of reasonable expenses
+
+We reimburse reasonable expenses incurred while working for the Foundation for everyone who works with us.
+
+We consider reasonable expenses to be:
+
+* costs incurred while working towards the [Foundation's mission](mission.md)
```
Comment (ElenaFdR) [inline organization/remuneration-policy.md]: Dutch original: voorgeschoten uitgaven voor het realiseren van het doel van de stichting)

**Comment by bvhme** [comment]
This merge is great, I think it is fine like this.

**Pull request merged**